worker/src/RTC/SimpleConsumer.cpp

```diff
@@ -457,6 +469,28 @@ namespace RTC
 		// Restore packet fields.
 		packet->SetSsrc(origSsrc);
 		packet->SetSequenceNumber(origSeq);
+
+		// If sent packet was the first packet of a key frame, we may need to send
+		// buffered packets belonging to the same key frame that arrived earlier due
+		// to packet misorder.
+		if (sendPacketsInTargetLayerRetransmissionBuffer)
+		{
+			for (auto& pair : this->targetLayerRetransmissionBuffer)
```

Comment:
Only those which seq number is higher than the key frame, right?

Reply:
Sure, I forgot. Done here: https://github.com/versatica/mediasoup/pull/1548/commits/38298856c6558515c830efc9c7f929cff6be0e42